csharp/src/Drivers/Apache/Hive2/HiveServer2Statement.cs

```diff
@@ -123,9 +138,23 @@ protected async ValueTask<Schema> GetSchemaAsync()
             return SchemaParser.GetArrowSchema(response.Schema);
         }
 
-        protected internal int PollTimeMilliseconds { get; } = PollTimeMillisecondsDefault;
+        protected internal int PollTimeMilliseconds { get; private set; } = PollTimeMillisecondsDefault;
+
+        protected internal int BatchSize { get; private set; } = BatchSizeDefault;
+
+        public static class StatementOptions
+        {
+            public const string PollTimeMilliseconds = "adbc.statement.polltime_milliseconds";
```

Comment:
Not sure about the name convention for these options. Should they be simple, like `polltime_milliseconds` instead of including a "namespace"?

Reply:
While we have examples of both (e.g. "password" vs "adbc.connection.readonly") I think the vast majority look like the latter.